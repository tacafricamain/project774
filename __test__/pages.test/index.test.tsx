import { render, screen } from '@testing-library/react'
import Home from '../../pages/index'


describe('Home', () => {
    it('renders a heading', () => {
    //   const { container } = 
    render(<Home />)
  
      const heading = screen.getByRole('heading', {
        name: /Digital Transformation To Remote Locations/i,
      })
  
      expect(heading).toBeInTheDocument()
  
    //   expect(container).toMatchSnapshot()
    })
  })
