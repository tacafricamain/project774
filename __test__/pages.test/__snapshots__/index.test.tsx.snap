// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <div
    class=""
  >
    <div
      class=" w-10/12 mx-auto"
    >
      <div
        class="flex flex-col lg:flex-row h-screen"
      >
        <div
          class="w-6/12 space-y-8 my-auto"
        >
          <h1
            class="text-black text-5xl font-semibold "
          >
            Digital Transformation 
            <br />
             To Remote Locations
          </h1>
          <p
            class="text-grayText"
          >
            Create an integrated and inclusive Digital
            <br />
            Society that improves the quality of 
            <br />
            life of Nigerian citizens in rural communities.
          </p>
          <div>
            <button
              class="text-white focus:ring-4 focus:outline-none
                                font-medium rounded-md text-sm px-8 py-3
                                text-center mr-2
                                undefined
                                bg-accentGreen text-white
                                "
              type="button"
            >
              Learn More
            </button>
            <button
              class="text-white focus:ring-4 focus:outline-none
                                font-medium rounded-md text-sm px-8 py-3
                                text-center mr-2
                                bg-transparent text-mainGreen border border-mainGreen
                                false
                                "
              type="button"
            >
              Our Vision
            </button>
          </div>
        </div>
        <div
          class="w-6/12"
        />
      </div>
    </div>
    <div
      class=" w-10/12 mx-auto my-20"
    >
      <div
        class="flex flex-col lg:flex-row h-48 "
      >
        <div
          class="w-6/12 "
        >
          <h1
            class="text-mainGreen text-3xl font-semibold text-center "
          >
            Get To know us 
            <hr
              class="w-40 h-1 bg-mainGreen inline-block"
            />
             
          </h1>
          <div />
        </div>
        <div
          class="w-6/12 "
        >
          <div
            class="w-10/12 mx-auto space-y-4"
          >
            <h2
              class="text-3xl font-semibold "
            >
              Introducing Project 774
            </h2>
            <p
              class="text-grayText"
            >
              To harness digital technologies and innovation to 
              <br />
              transform remote communities to promote inclusive 
              <br />
              economic growth for socio-economic development.
            </p>
            <button
              class="text-white focus:ring-4 focus:outline-none
                                font-medium rounded-md text-sm px-8 py-3
                                text-center mr-2
                                undefined
                                bg-accentGreen text-white
                                "
              type="button"
            >
              Learn More
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="flex flex-col lg:flex-row h-screen bg-backgroundGrey"
    >
      <div
        class=" w-10/12 mx-auto my-8"
      >
        <div
          class="w-6/12 space-y-8 flex flex-col h-full justify-center"
        >
          <h1
            class="text-black text-4xl font-semibold "
          >
            Building up Communities 
            <br />
             that thrives on Innovation.
          </h1>
          <p
            class="text-grayText"
          >
            Project 774 aims to spark interest and curiosity 
            <br />
            across all local governments of the Nigerian 
            <br />
              society thereby
          </p>
          <ul
            class="space-y-4 text-grayText"
          >
            <li
              class="inline-block"
            >
              <span
                class="rounded-full bg-mainGreen w-5 h-5"
              >
                <svg
                  class="flex-shrink-0 w-5 h-3 text-white inline"
                  fill="currentColor"
                  viewBox="0 0 20 20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
               Increase adoption of the e-naira digital currency.
            </li>
            <li
              class="inline-block"
            >
              <span
                class="rounded-full bg-mainGreen w-5 h-5"
              >
                <svg
                  class="flex-shrink-0 w-5 h-3 text-white inline"
                  fill="currentColor"
                  viewBox="0 0 20 20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
               Create employment opportunities for digital jobs.
            </li>
            <li
              class="inline-block"
            >
              <span
                class="rounded-full bg-mainGreen w-5 h-5"
              >
                <svg
                  class="flex-shrink-0 w-5 h-3 text-white inline"
                  fill="currentColor"
                  viewBox="0 0 20 20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
               Increase adoption of the e-naira digital currency.
            </li>
          </ul>
          <div />
        </div>
        <div
          class="w-6/12"
        />
      </div>
    </div>
  </div>
</div>
`;
